docs/StarCellBio-Architecture.rst

```diff
@@ -75,6 +75,11 @@ The top level object is a list of assignments, represented by
 defined by common list code (described later). Each element of the list is 
 represented by Assignment object as described in Assignments.js.
 
+.. image:: objectDiagram.svg
```

Comment:
This appears to be due to mime-type happening on github since it is assuming you want to look at an SVG as XML.  I think it would be easiest to add a rendered copy to, but still have it link to the svg.